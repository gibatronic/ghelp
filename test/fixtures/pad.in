# generate build
build: scripts styles

# generate scripts
scripts:
	echo scripts

# generate styles
styles:
	echo styles
