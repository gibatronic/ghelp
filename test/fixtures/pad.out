  build    generate build
  scripts  generate scripts
  styles   generate styles
